prereise/gather/solardata/nsrdb/sam.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import pandas as pd
 import PySAM.Pvwattsv7 as PVWatts
-import PySAM.PySSC as pssc
+import PySAM.PySSC as pssc  # noqa: N813
```

Comment:
What is the issue here?

Reply:
We get the following warning:
`prereise/gather/solardata/nsrdb/sam.py:4:2: N813 camelcase 'PySAM.PySSC' imported as lowercase 'pssc'`